packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  Component,
+  Event,
+  EventEmitter,
+  Fragment,
+  Prop,
+  h,
+} from '@stencil/core';
+
+@Component({
+  tag: 'va-telephone',
+  styleUrl: 'va-telephone.css',
+  shadow: true,
+})
+export class VaTelephone {
+  /**
+   * 3 or 10 digit string representing the contact number
+   */
+  @Prop() contact!: string;
+
+  /**
+   * Optional phone number extension
+   */
+  @Prop() extension: number;
+
+  /**
+   * Indicates if the phone number can be clicked or not
+   */
+  @Prop() notClickable: boolean = false;
+
+  /**
+   * Indicates if this is a number meant to be called from outside the US.
+   * Prepends a "+1" to the formatted number.
+   */
+  @Prop() international: boolean = false;
+
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
```

Comment:
This is completely new. The React component had an `onClick` prop that [would get set in the component](https://github.com/department-of-veterans-affairs/component-library/blob/4a7c522fe2f21d8ba0e7b0bea795fe0d959a2a0c/packages/react-components/src/components/Telephone/Telephone.jsx#L170). This is only used in one file, and it is exclusively for analytics event tracking.

![image](https://user-images.githubusercontent.com/2008881/144945277-10f6a1ba-0bb8-4282-9572-7a0a9f11a7a4.png)


https://github.com/department-of-veterans-affairs/vets-website/blob/74b6c6bdc4fe99f55a41d9dc3db7eef58b118e67/src/applications/claims-status/components/StemAskVAQuestions.jsx#L11-L16